### ISSUE TITLE
Navbar Tag's , problems with accentuation.

### ISSUE THREAD

**Issue by nelsonnovais** [reactioncommerce/reaction#2409]
### Expected behavior
It is expected when Admin is creating a new Tag for a given type of product to be able create a tag with accentuation for a specific language, like "Relógio" (watch) or "Algodão" (cotton)

### Actual Behavior
But actually what happens when some user tries to create some tag like "Relógio", it end up like 
"Rel´ogio" or " Algod~ao".

### Steps to Reproduce the Behavior
Go to administration, log in, create a new tag for a given product, try to input some word with accentuation.

### Versions
Node: 8.0.0
NPM: 5.0.0
Meteor Node: 4.8.3
Meteor NPM: 4.6.1
Reaction CLI: 0.9.2
Reaction: 1.2.0
Docker: 17.03.1-ce

**Comment by aaronjudd** [comment]
For me, `Relógio` ends up as `http://localhost:3000/reaction/tag/relogio` or "Algodão" 
 as `http://localhost:3000/reaction/tag/algodao` and "Relógio Algodão" ends up as `http://localhost:3000/reaction/tag/relogio-algodao`.  Is this not the result you are seeing?  (mind I copied and pasted, maybe it's different when manually entered?)

**Comment by aaronjudd** [comment]
Also note, tested in the `development` branch, which has an updated version of https://github.com/andyhu/transliteration

```
Node: 7.5.0
NPM: 5.0.0
Meteor Node: 4.8.2
Meteor NPM: 4.5.0
Reaction CLI: 0.9.3
Reaction: 1.2.0
Reaction branch: marketplace-shop-switcher
Docker: 17.03.1-ce
```

**Comment by nelsonnovais** [comment]
@aaronjudd You'r right, if tou copy past it, it ends up like "http://localhost:3000/lana/tag/relogio", but if you type it, the result is: "http://localhost:3000/lana/tag/rel-ogio".
See image:
![image](https://user-images.githubusercontent.com/28451402/27075245-7c9e1462-5021-11e7-93c8-f22ea5331a7b.png)

But, strangely in the product tags, this behavior does not occur.

**Comment by cusspvz** [comment]
@aaronjudd I didn't reproduced it, but it seems to me this issue could be linked to some method listening on each keystroke?

**Comment by efalayi** [comment]
@aaronjudd `Reaction.getSlug(tagName)` sets the slug for a new tag. I checked through the codebase and discovered that `getSlug` function in `/lib/api/helpers.js` uses `transliteration` package. This package kinda of translates the string parsed as a parameter. I tried setting the slug to the `tagName` and this was reflected in the tag url.
I also did a quick search on (transliteration) [https://github.com/ongoworks/transliteration]: it's been deprecated. Only useful with Meteor versions prior to < 1.4.
